cluster/swarm/cluster.go

```diff
@@ -310,6 +311,11 @@ func (c *Cluster) Info() [][2]string {
 		info = append(info, [2]string{" └ Containers", fmt.Sprintf("%d", len(engine.Containers()))})
 		info = append(info, [2]string{" └ Reserved CPUs", fmt.Sprintf("%d / %d", engine.UsedCpus(), engine.TotalCpus())})
 		info = append(info, [2]string{" └ Reserved Memory", fmt.Sprintf("%s / %s", units.BytesSize(float64(engine.UsedMemory())), units.BytesSize(float64(engine.TotalMemory())))})
+		labels := make([]string, 0, len(engine.Labels))
+		for k, v := range engine.Labels {
+			labels = append(labels, k+"="+v)
+		}
```

Comment:
Would be great if you can add `sort.Strings(labels)` here to have a sorted output for better readability (with a lot of entries it can be difficult to locate a label)